packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.workspace.tsx

```diff
@@ -314,6 +320,28 @@ const ImmunizationsForm: React.FC<DefaultPatientWorkspaceProps> = ({
               </ResponsiveWrapper>
             </section>
           )}
+          <section>
+            <ResponsiveWrapper>
+              <Controller
+                name="note"
+                control={control}
+                render={({ field: { onChange, value } }) => (
+                  <div className={styles.row}>
+                    <TextArea
+                      enableCounter
+                      id="note"
+                      invalidText={errors?.note?.message}
+                      labelText={t('note', 'Note')}
+                      placeholder={t('immunizationNotePlaceholder', 'E.g. mild redness at injection site')}
```

Comment:
Per this [recent audit](https://docs.google.com/spreadsheets/d/1AnMqv80Pb9CI8FJqNnoNNh9p5O-CdVCl7zB5BY4m8wQ/edit?pli=1&gid=0#gid=0) of our translations, the abbreviation "E.g." is discouraged because it may not be universally understood across different languages and regions. Using the full phrase "For example" provides clearer context for translators and follows i18n best practices.

```suggestion
                      placeholder={t('immunizationNotePlaceholder', 'For example: mild redness at injection site')}
```